client/blocks/login/index.jsx

```diff
@@ -405,6 +411,7 @@ class Login extends Component {
 
 export default connect(
 	state => ( {
+		closeWindowAfterLogin: getCloseWindowAfterLogin( state ),
```

Comment:
I wonder if we need to introduce a new selector. Could we use the existing `getCurrentQueryArguments` selector instead? Would make this PR simpler.